tests/jenkins/TestMavenSignReleaseJob.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import jenkins.tests.BuildPipelineTest
+import org.junit.Before
+import org.junit.Test
+
+class TestMavenSignReleaseJob extends BuildPipelineTest {
+
+    @Before
+    void setUp() {
+
+        def localPath = "${this.workspace}/artifacts"
+
+        def artifactsPath = "${this.workspace}/artifacts/distribution-build-opensearch/1.0.0/123/linux/x64/builds/opensearch/manifest.yml"
+
+        def bucketPath = 'distribution-build-opensearch/1.0.0/123/linux/x64/builds'
```

Comment:
I think this is the `artifactPath` right? We could use this variable to set the values on line 24 and 33

Reply:
Good point!